docs/UserGuide.adoc

```diff
@@ -67,105 +72,112 @@ Examples:
 
 * `help`
 
-* `man add`
+* `man task`
+
+=== Adding a task: `task`
+
+Adds a task to the board +
+Format: `add n/NAME d/DATETIME [r/REMARK]`
+
+Examples:
 
-=== Adding a person: `add`
+* `task n/Software Engineering Milestone 1 d/16/3/2018 17:00 r/urgent`
+* `task n/Programming Methodology Assignment 2 d/16/3/2018 23:59`
 
-Adds a person to the address book +
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...`
+=== Adding an event: `event`
 
-[TIP]
-A person can have any number of tags (including 0)
+Adds a event to the board +
+Format: `event n/NAME s/STARTTIME e/ENDTTIME [l/LOCATION] [r/REMARK]`
 
 Examples:
 
-* `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
-* `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
+* `event n/Software Project s/1/5/2018 8:00 e/1/8/2018 8:00 l/School of Computing r/remember to bring laptop charger`
+* `event n/Blockchain Talk s/16/3/2018 16:00 e/16/3/2018 18:00`
 
-=== Listing all persons : `list`
+=== Listing all uncompleted tasks: `ls task`
 
-Shows a list of all persons in the address book. +
-Format: `list`
+Shows a list of all uncompleted tasks in the board. +
+Format: `ls task`
 
-=== Editing a person : `edit`
+=== Listing all uncompleted events: `ls event`
 
-Edits an existing person in the address book. +
-Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]...`
+Shows a list of all uncompleted event in the board. +
+Format: `ls event`
 
-****
-* Edits the person at the specified `INDEX`. The index refers to the index number shown in the last person listing. The index *must be a positive integer* 1, 2, 3, ...
-* At least one of the optional fields must be provided.
-* Existing values will be updated to the input values.
-* When editing tags, the existing tags of the person will be removed i.e adding of tags is not cumulative.
-* You can remove all the person's tags by typing `t/` without specifying any tags after it.
-****
+=== Editing a person : `edit task` *`Coming Soon in v2.0`*
 
-Examples:
+Edits task in the board. +
+Format: `edit task INDEX`
```

Comment:
include edit event command also?

Reply:
Not sure what are the requirements, but I decided not to implement edit/find commands for v1.5